Changes for 11.6 Onsite (#2591)
SuperOfficeDocs/superoffice-docs: docs/en/onsite/includes/warn-dbsetup-discontinued.md

<!-- markdownlint-disable-file MD041 -->
> [!WARNING]

> >

Reviewer: ```suggestion
```

No blank line here

> > Since SuperOffice 11.6 ServerSetup.exe and DBSetup.exe has been discontinued and replaced by CRMSetup.exe and CRMTask.exe command line utilities.

Reviewer: ```suggestion
> From SuperOffice 11.6, ServerSetup.exe and DBSetup.exe have been discontinued and replaced by the CRMSetup.exe and CRMTask.exe command-line utilities.
```
